voting_schemes/electionguard/python-wrapper/src/bulletin_board/electionguard/common.py

```diff
@@ -126,16 +149,17 @@ def process_message(self, message_type: str, message: Content) -> Content:
         return responses
 
     def is_fresh(self) -> bool:
-        return isinstance(self.step, self.starting_step)
+        return type(self.step) is self.__class__.starting_step
```

Comment:
💬 Maybe we could make this method more explicitly. I think this should be only in the Trustee wrapper and we could set a variable to `False` after sending the election keys. What do you think?

Reply:
I'm not sure about making it concrete to the Trustee. I think this is a general question you can ask about any wrapper --is it fresh or has it been restored from some backup?